src/status_im2/contexts/share/events.cljs

```diff
@@ -0,0 +1,24 @@
+(ns status-im2.contexts.share.events
+  (:require [utils.re-frame :as rf]
+            [status-im2.common.toasts.events :as toasts]
+            [quo2.foundations.colors :as colors]))
+
+(rf/defn open-profile-share-view
+  {:events [:share/open]}
+  [{:keys [db]}]
+  {:dispatch [:show-popover
```

Comment:
`:dispatch` shouldn't be used inside event handlers

Reply:
instead of `:share/open`  m `:open-modal` should be used

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/c85f47dcf261fb4e101fdb75ff9d8b7ccb34970f